tests/integration/conftest.py

```diff
@@ -52,6 +52,10 @@ def salt_call_cli(minion):
 def integration_test_config():
     default_path = Path().parent.parent / "local" / "vcenter.conf"
     config_path = os.environ.get("VCENTER_CONFIG", default_path)
+
+    if isinstance(config_path, str):
+        config_path = Path(config_path)
```

Comment:
**suggestion** since os.environ.get always returns a string
```
>>> import os
>>> import pathlib
>>> os.environ['NOPE'] = pathlib.Path()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.9/os.py", line 684, in __setitem__
    value = self.encodevalue(value)
  File "/usr/lib/python3.9/os.py", line 756, in encode
    raise TypeError("str expected, not %s" % type(value).__name__)
TypeError: str expected, not PosixPath
```
And `Path(Path())` is entirely valid
```
>>> pathlib.Path(pathlib.Path())
PosixPath('.')
```
skip the instance check: `config_path = Path(os.environ.get(...))`

Reply:
done